RecoPixelVertexing/PixelVertexFinding/src/gpuVertexFinder.cc

```diff
@@ -0,0 +1,9 @@
+#define CUDA_KERNELS_ON_CPU // not needed yet
+#include "gpuVertexFinderImpl.h"
+namespace gpuVertexFinder {
+template<>
```

Comment:
I do not exclude that one can avoid this indirection and maintain the single source implementation
with double definition